semsim/compute_pairwise_similarities.py

```diff
@@ -69,9 +67,15 @@ def compute_pairwise_sims(
             list(rs_df["destination"]),
         )
 
+        # Remap node IDs to node names
+        node_map = dag.get_nodes_mapping()
+        id_map = {v: k for k, v in node_map.items()}
+        for col in ['source', 'destination']:
+            rs_df = rs_df.replace({col: id_map})
+
         print(f"Writing output to {rs_path}...")
         rs_df.sort_values(
-            by=["similarity"], ascending=False, inplace=True
+            by=["resnik_score"], ascending=False, inplace=True
```

Comment:
I am still very against this design... :P Not your fault, need to hash this out with @cmungall. Having the a generic `score` column is much more flexible than encoding all scores in different columns

Reply:
I like to know what kind of score it is. Could we include a `score_types` column or something?

Reply:
In OAK you dont have a choice. Whatever the semsim model says... :/